lib/core/src/Cardano/Wallet/Transaction.hs

```diff
@@ -255,34 +257,28 @@ data TransactionLayer k tx = TransactionLayer
         --
         -- Returns `Nothing` for ByronEra transactions.
 
-    , distributeSurplus
+    , minimizeFee
```

Comment:
This function (and its subordinate functions) should be changed back to `distributeSurplus`.